contrib/syncer/README.md

```diff
@@ -0,0 +1,31 @@
+# KCP: Install Syncer component on the P-cluster
+
+
+1. Create workspace on your Kcp cluster (remember to use your Kcp credentials)
+
+```sh
+cat <<EOF | kubectl --kubeconfig .kcp/admin.kubeconfig apply -f -
+apiVersion: tenancy.kcp.dev/v1beta1
+kind: Workspace
+metadata:
+  name: my-workspace
+spec:
+  type: Universal
+EOF
+```
+
+We can see it has been created correctly and is ready:
+
+```sh
+kubectl get workspaces my-workspace -o wide
+NAME           TYPE        PHASE
+my-workspace   Universal   
+```
+
+2. ** FIXME ** Create a service account on the workspace 
```

Comment:
Is it necessary to create a new service account in the namespace vs just using the default one that comes with a namespace?

Reply:
It's probably better to have a new, dedicated SA just for the syncer, isn't it?

Reply:
+1